apps/api/scheduler.sh

```
@@ -0,0 +1,18 @@
+
+SCHEDULE_TIME=1
+
+source ".env"
+
+BASE_URL="http://localhost:${SERVER_PORT}/summary"
+
+add_pending_summaries_to_queue() {
+  curl -X POST "${BASE_URL}/schedule" -H "Content-Type: application/json" -d '{}'
+}
+
+last_archive_run=$(date +%s)
```

Shellcheck (0.10.0)

[warning] last_archive_run appears unused. Verify use (or export if used externally).

(SC2034)